### PR TITLE
Improve context menu action configs and UX

### LINKED ISSUE #151: Improve Loadout Editing UX in a unit’s context menu
**Is your feature request related to a problem?**
As a game master I would like to quickly access the (ACE) Arsenal UI for a unit.

**Solution you'd like:**
Add an option to the context menu when right clicking a unit to open the (ACE) Arsenal for said unit. (e.g. as a child entry in "Edit Loadout")

**Alternatives you've considered:**
A configurable modifier key while clicking a unit could be a viable alternative.

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- Cleanup and improve `context_actions` component
    - Improve loadout/inventory/vehicle appearance actions UX by adding a "Edit" sub-action but keeping main action as a shortcut, Close #151 
    - Move code from config to separate functions - most are very short but I think it is better than keeping in config
    - Add combat mode action (also improve other group property action conditions to exclude player only groups)
    - Add icons for actions that didn't previously have them
    - Move remote control action to `context_actions` component
    - Cleanup `priority` entry usage, requires CBATeam/CBA_A3#1271
- Improve right-click context menu opening condition check - allow opening if groups of `sideLogic` are selected
- Simplify context menu local scope variable names, allow `_args` to be defined in config

### CODE REVIEW

File: addons/context_actions/functions/fnc_canEditInventory.sqf [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ *
+ * Example:
+ * [_entity] call zen_context_actions_fnc_canEditInventory
+ *
+ * Public: No
+ */
+
+params ["_entity"];
+
+_entity isEqualType objNull && {alive _entity} && {getNumber (configFile >> "CfgVehicles" >> typeOf _entity >> "maximumLoad") > 0}
```
Comment (CreepPork) [inline addons/context_actions/functions/fnc_canEditInventory.sqf]: Maybe push each and check into a new line? E.g.:
```suggestion
_entity isEqualType objNull
    && {alive _entity}
    && {getNumber (configFile >> "CfgVehicles" >> typeOf _entity >> "maximumLoad") > 0}
```

I'm not certain how the whole project manages them but I think these context_action functions would benefit from them to being more quickly readable without just being a long line of &&'s.

File: addons/context_actions/functions/fnc_teleportPlayers.sqf [hunk 2]
```diff
@@ -23,7 +23,7 @@ private _players = _this select {isPlayer _x};
     if (_successful) then {
         curatorMouseOver params ["_type", "_entity"];
 
-        if (_type isEqualTo "OBJECT" && {_entity isKindOf "AllVehicles"} && {!(_entity isKindOf "CAManBase")}) then {
+        if (_type == "OBJECT" && {_entity isKindOf "AllVehicles"} && {!(_entity isKindOf "CAManBase")}) then {
```
Comment (CreepPork) [inline addons/context_actions/functions/fnc_teleportPlayers.sqf]: Why remove the `isEqualTo`?

Comment (mharis001) [reply addons/context_actions/functions/fnc_teleportPlayers.sqf]: Either works here, changed to match other `curatorMouseOver`'s type checks.

File: addons/context_actions/functions/fnc_canEditInventory.sqf (hunk 1 above)
Comment (mharis001) [reply addons/context_actions/functions/fnc_canEditInventory.sqf]: No real rule, I generally multi-line if there are more than 3 or 4 conditions.

**Comment by mharis001** [comment]
Will merge after CBA 3.13.1 releases.

**Pull request merged**